autometa/config/default.config

```diff
@@ -60,6 +61,9 @@ bacteria_single_copy = https://${markers:host}/KwanLab/Autometa/main/autometa/da
 bacteria_single_copy_cutoffs = https://${markers:host}/KwanLab/Autometa/main/autometa/databases/markers/bacteria.single_copy.cutoffs
 archaea_single_copy = https://${markers:host}/KwanLab/Autometa/main/autometa/databases/markers/archaea.single_copy.hmm
 archaea_single_copy_cutoffs = https://${markers:host}/KwanLab/Autometa/main/autometa/databases/markers/archaea.single_copy.cutoffs
+proteins_aa_reps = https://${gtdb:host}/releases/${gtdb:release}/genomic_files_reps/gtdb_proteins_aa_reps.tar.gz
+gtdb_taxdmp = https://${markers:host}/shenwei356/gtdb-taxdump/releases/latest/download/gtdb-taxdump.tar.gz
```

Comment:
This probably should not be referencing markers. Kind of confusing. The full Github path should probably be used. Also, this should either specify `taxonkit` or be generalized by some other selection to use `taxonkit` or another tool

Reply:
```suggestion
taxonkit_gtdb_taxdump = https://raw.githubusercontent.com/shenwei356/gtdb-taxdump/releases/latest/download/gtdb-taxdump.tar.gz
```